platform/core/src/services/ToolBarService/ToolBarService.ts

```diff
@@ -52,19 +52,32 @@ export default class ToolBarService {
     this.buttons = {};
   }
 
+  onModeEnter() {
+    this.reset();
+  }
+
   /**
    *
-   * @param {*} interaction
+   * @param {*} interaction - can be undefined to run nothing
+   * @param {*} extraOptions to include in the commands run
    */
-  recordInteraction(interaction) {
+  recordInteraction(interaction, extraOptions) {
```

Comment:
can we call this options? instead of extraoptions?

Reply:
It is extra options because it is in addition to the default options specified by the commands.